en/docs/Compliance/general-Data-Protection-Regulation.md

```diff
@@ -197,44 +168,37 @@ Following features are supported as part of Self-care User Portal:
     processing activities carried out by the organization, their
     purposes, and time-limits and what data are stored can be made
     transparent to users via the IS Self-care User Portal.  
-      
+<br/>     
 -   **The right of access -** Via the IS Self-care User Portal, users
     can access and review what personal data are stored in the
     processing organization.  
-    **  
-    **
+<br/>
 -   **The right to rectification -** Individuals can rectify incorrect
     data on their user profiles by themselves by logging into Self-care
     User Portal.  
-    **  
-    **
+<br/>
 -   **The right to restrict processing -** Individuals can make
     restrictions on their user profiles by themselves by logging into
```

Comment:
logging into --> logging in to

Reply:
Fixed with commit 895dc3b5173232dd1a2cd561fb9d5b264bb096bb